dev/Breadcrumb/BreadcrumbBar.xaml

```diff
@@ -25,9 +25,8 @@
             <Setter.Value>
                 <ControlTemplate TargetType="local:BreadcrumbBarItem">
                     <Grid x:Name="PART_LayoutRoot"
-                          Margin="{StaticResource BreadcrumbBarItemDefaultMargin}"
```

Comment:
Is there a reason margin and padding is removed? Should be be template bound so that it is publically settable?